### PR TITLE
Add Azure Function and remove Validation and Webhook activities

### PR DESCRIPTION
**Pull request by arsunda**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/DataFactory/Management.DataFactory/Generated/Models/ValidationActivity.cs [hunk 1]
```diff
@@ -1,127 +0,0 @@
...
-        /// <summary>
-        /// Gets or sets specifies the timeout for the activity to run. If
-        /// there is no value specified, it takes the value of
-        /// TimeSpan.FromDays(7) which is 1 week as default. Type: string (or
-        /// Expression with resultType string), pattern:
-        /// ((\d+)\.)?(\d\d):(60|([0-5][0-9])):(60|([0-5][0-9])). Type: string
-        /// (or Expression with resultType string), pattern:
-        /// ((\d+)\.)?(\d\d):(60|([0-5][0-9])):(60|([0-5][0-9])).
-        /// </summary>
-        [JsonProperty(PropertyName = "typeProperties.timeout")]
```
Comment (dsgouda) [inline src/SDKs/DataFactory/Management.DataFactory/Generated/Models/ValidationActivity.cs]: Please note that this is a breaking change

File: src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj [hunk 2]
```diff
@@ -10,17 +10,18 @@
 		<AssemblyName>Microsoft.Azure.Management.DataFactory</AssemblyName>
 		<PackageTags>Microsoft Azure resource management;Data Factory;ADF;</PackageTags>
 		<PackageReleaseNotes>	<![CDATA[
-        - Added table name in dataset
-		- Added get Dataplane access API
```
Comment (dsgouda) [inline src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj]: Just to be sure, are you planning to bump the version in a different PR

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part, please revert to the questions.

File: src/SDKs/DataFactory/Management.DataFactory/Generated/Models/ValidationActivity.cs (hunk 1 above)
Comment (hvermis) [reply src/SDKs/DataFactory/Management.DataFactory/Generated/Models/ValidationActivity.cs]: @dsgouda  This changes hasn't been released yet. We had similar comments on Specs PR and got sign off from them and ARM team. These are new activities we added last week, but didn't want them in this release, so we took them out

File: src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj (hunk 2 above)
Comment (hvermis) [reply src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj]: The version is already bumped by Meiyan in her PR to 3.0.0. That has not been released yet.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**